_15thnight/__init__.py

```diff
@@ -53,194 +60,40 @@ def shutdown_session(response):
-        return render_template(
-            'dashboard/provider.html',
-            user=current_user,
-            alerts=Alert.get_active_alerts_for_provider(current_user)
-        )
-
-
-@app.route('/delete_user', methods=['POST'])
-@login_required
-def delete_user():
-    if current_user.role != 'admin':
-        flash('Access denied', 'danger')
-        return redirect(url_for('dashboard'))
-    form = DeleteUserForm()
-    if form.validate_on_submit():
-        user = User.get(form.id.data)
-        user.delete()
-        flash('User Deleted Successfully', 'success')
-    else:
-        flash('Failed to delete user', 'danger')
-    session['deleted_user'] = True
-    return redirect(url_for('dashboard'))
-
-
-@app.route("/logout")
-@login_required
+@app.route('/<path:path>')
+def index(path=None):
+    """Serve the index"""
+    state = json.dumps(dict(
+        current_user=current_user if current_user.is_authenticated else None
+    ))
+    return render_template('index.html', state=state)
+
+@app.route('/logout')
 def logout():
-    """User logout."""
-    session.clear()
-    flash('You have been logged out!', 'success')
-    return redirect(url_for('index'))
-
+    logout_user()
+    return redirect('/')
 
 @app.route('/health')
 def healthcheck():
     """Low overhead health check."""
     return 'ok', 200
 
 
-@app.route('/about')
-def about():
-    """Simple about page route."""
-    return render_template('about.html')
-
-
-@app.route('/contact', methods=['GET', 'POST'])
-def contact():
-    if request.method == 'POST':
-        flash('you tried to make a post')
-        email = request.form['email']
-        message = request.form['message']
-        send_email(to=email, subject="Contact Form", body=message)
-        return redirect(url_for('login'))
-    return render_template('contact.html')
-
-
-@app.route('/respond_to/<int:alert_id>', methods=['GET', 'POST'])
-@login_required
-def response_submitted(alert_id):
-    """
-    Action performed when a response is provided.
-
-    Text the creator of the alert:
-        - email, phone, and things able to help with of the responding user.
-    """
-    if request.method == 'POST':
-        submitted_message = request.form['message']
-        responding_user = current_user
-        try:
-            alert = Alert.query.get(int(alert_id))
-        except Exception as e:
-            return 'Error {}'.format(e), 404
-
-        respond_to_alert(responding_user, submitted_message, alert)
-
-        flash(
-            'Your response has been sent to the advocate, thank you!',
-            'success'
-        )
-        return redirect(url_for('dashboard'))
-    else:
-        try:
-            alert = Alert.query.get(int(alert_id))
-        except Exception as e:
-            return 'Error {}'.format(e), 404
-
-        return render_template(
-            'respond_to.html',
-            alert=alert, user=current_user, form=ResponseForm()
-        )
-
 if __name__ == '__main__':
```

Comment:
if __name__ == '__main__':

and below should probably be removed. We do not suggest, nor use the app in the manner.